helper-script

```diff
@@ -0,0 +1,270 @@
+#!/bin/sh
+
+# URLs
+readonly steamcmdurl="https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz"
+
+# dirs
+readonly XDG_CACHE_HOME=${XDG_CACHE_HOME:-$HOME/.cache}
+readonly XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+scriptdir=$(dirname "$0")
+
+# variables
+ats=false
+ets2=false
+update=false
+startgame=false
+
+# find latest proton folder
+find_proton() {
```

Comment:
I will think about this again.
We can't tell in which `$STEAMLIBRARY` the proton folder is so we have to scan them all. They're listed in `$XDG_DATA_HOME/Steam/SteamApps/libraryfolders.vdf`. Also there can be multiple versions installed at once because steam doesn't delete the old ones.

Reply:
I've finally setup proton on my computer, and it seems there are some variations: my `libraryfolders.vdf` doesn't contain any path, even after starting a game with proton.
Also small detail: the path in my installation is `$XDG_DATA_HOME/Steam/steamapps/` not `SteamApps`

Reply:
Your `libraryfolder.vdf` only contains a path if you've set up additional libraries before.
Maybe your installation ist newer and they've renamed the default folder… This worked out in a quick test: `$XDG_DATA_HOME/Steam/[Ss]team[Aa]pps/common`

Reply:
Let's drop this entirely and just rely on user input or our own proton since we're using steamcmd anyway. 300MB more to download if the user is lazy but that's nothing compared to the actual game.